src/status_im2/contexts/chat/composer/link_preview/events_test.cljs

```diff
@@ -0,0 +1,212 @@
+(ns status-im2.contexts.chat.composer.link-preview.events-test
```

Comment:
Hey @erikseppanen, I had a lot of success writing tests for events in this way, without stubbing effects like I did for the Activity Center. These ones caught more issues during development and they were more useful and simpler to write. So this is my new favorite way to test event handlers, just as normal pure functions. It was a fun ride ;)